frontend/src/fetchers/ssdi.js

```diff
@@ -36,14 +72,14 @@ export const getSelfServiceDataQueryString = (filterName, filters) => {
 
 const getSelfServiceUrl = (filterName, filters) => {
   const queryString = getSelfServiceDataQueryString(filterName, filters);
-  const baseUrl = join(ssdiUrl, filterName);
+  const baseUrl = join(urlsForQuery[filterName]);
   return `${baseUrl}?${queryString}`;
 };
 
-export const getSelfServiceData = async (filterName, filters) => {
+export const getSelfServiceData = async (filterName, filters, dataSetSelection) => {
   const url = getSelfServiceUrl(filterName, filters);
-
-  const response = await get(url);
+  const urlToUse = url + (dataSetSelection && dataSetSelection.length ? dataSetSelection.map((s) => `&dataSetSelection[]=${s}`).join('') : '');
```

Comment:
do we need to account for this case (dataSetSelection is falsy?) If so, did we add a test.
Note also there is no need to check for length: if dataSetSelection is empty, the map will be an empty array and will join out to be the same as your else statement

Reply:
Removed checks.